File: python/bubble_sort.py
from distutils.dep_util import newer_pairwise
from multiprocessing.dummy import current_process


sequence = [4, 3, 5, 0, 1]
swaps = 0
new_sequence = sequence.copy()
# print(new_sequence)
# Your Code Here
def bubbleSort(new_sequence):
    for x in range(0, len(sequence), 1):
        for y in range(1, len(sequence), 1):
            if new_sequence[x] > new_sequence[y]:
                swaps += 1
                previous = new_sequence[x]
                current = new_sequence[y]
                previous = current
                current = previous
    print(new_sequence)

bubbleSort(new_sequence)

File: js/bubbleSort.js
var sequence = [5,3,0,4,1]
var swaps = 0
let newSwquence = sequence.slice()
// Your Code Here
function bubbleSort(arr){
    for (let i = 0; i < newSwquence.length; i++) {
        for (let j = i + 1; j < newSwquence.length; j++) {
            if (newSwquence[i] > newSwquence[j]) {
                swaps += 1
                let previous = newSwquence[i]
                let current = newSwquence[j]
                let x = newSwquence[i] = current
                let y =newSwquence[j] = previous
                console.log(x, y)
            }
        }
    } return newSwquence
    
}

console.log(bubbleSort(newSwquence))
console.log("Final result: " + newSwquence)
console.log("Swaps: " + swaps)


